app/models/book.rb

```diff
@@ -18,4 +18,15 @@ class Book < ApplicationRecord
   scope :featured, -> { where(featured: true) }
 
   paginates_per 3
+
+  def self.to_csv
+    attributes = %w{ name isbn featured }
+    CSV.generate() do |csv|
```

Comment:
No need for the () at the end of `generate`. By convention in Ruby, we only provide brackets when we are passing arguments through.

The only exception to this rule is the `super` method, but we'll get into that a little ways down the track.